text/0000-add-ansi-escape-codes.md

```diff
@@ -0,0 +1,88 @@
+- Feature Name: add_ansi_escape_codes
+- Start Date: 2022-02-14
+- RFC PR: (leave this empty)
+- Pony Issue: (leave this empty)
+
+# Summary
+
+This RFC proposes the addition of more escape code functions to the ANSI
+primitive of the term package.
+
+# Motivation
+
+The purpose of these additional methods is to cover more of the standard ANSI
+escape codes within the stdlib.
+
+# Detailed design
+
+## Added Escape Codes
+
+- 8-bit forground `\x1B[38;5;nm` and background `\x1B[48;5;nm` colors
+- 24-bit foreground `\x1B[38;2;r;g;bm` and background `\x1B[48;2;r;g;bm` colors
+- enter alternate screen buffer `\x1B[?1049h` and leave alternate screen buffer `\x1B[?1049l`
+- bell alert `\x7`
+- scroll up `\x1B[nS` and scroll down `\x1B[nT`
+- next line `\x1B[nE` and previous line `\x1B[nF`
+- cursor horizontal absolute `\x1B[nG`
+- hide cursor `\x1B[?25l` and show cursor `\x1B[?25h`
+- save cursor position `\x1B[s` and restore cursor position `\x1B[u`
+- device status report `\x1B[6n` (sends cursor position to stdin)
+- faint `\x1B[2m`, italic `\x1B[3m`, conceal `\x1B[8m`, and strike `\x1B[9m` text
+- erase in display `\x1B[nJ`
+- erase in line  `\x1B[nK`
+
+
+## Erase In Display/Line
+
+There are currently two similar functions, but they have a couple of issues:
+- Neither of them expose the parameter that the escape codes take.  
+- The `clear` function moves the cursor to the top left and then clears the screen.  
+This may have been done to create the same result that this escape code gave on DOS.
```

Comment:
```suggestion
- The `clear` function moves the cursor to the top left and then clears the screen.  

This may have been done to create the same result that this escape code gave on DOS.
```

Reply:
If that extra newline is added, the next line would be split out into a separate paragraph instead of being attached to the bullet point about the functionality of the current clear function.